### PR TITLE
EPMEDU-752: Align "More" page with new design

### PR DESCRIPTION
**Pull request by Fileek**

### CODE REVIEW

**Comment by Fileek** [comment]
https://user-images.githubusercontent.com/83027107/176613302-7d38b675-bd54-4323-bced-14ca752e8890.mp4

File: base/src/main/java/com/epmedu/animeal/base/ui/ShortButton.kt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        Text(text = text)
+    }
+}
+
+@Preview
+@Composable
+fun ShortButtonPreview() {
+    AnimealTheme {
+        Surface {
+            ShortButton(text = "Click", onClick = {})
```
Comment (egorikftp) [inline base/src/main/java/com/epmedu/animeal/base/ui/ShortButton.kt]: please add preview for disabled state

**Comment by Fileek** [comment]
https://user-images.githubusercontent.com/83027107/176650922-809c8e85-e306-4ff1-b4a6-756e250c6b1f.mp4

File: base/src/main/java/com/epmedu/animeal/base/ui/ShortButton.kt (hunk 1 above)
Comment (Fileek) [reply base/src/main/java/com/epmedu/animeal/base/ui/ShortButton.kt]: Done

File: base/src/main/java/com/epmedu/animeal/base/ui/ShortButton.kt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import androidx.compose.ui.unit.dp
+import com.epmedu.animeal.base.theme.AnimealTheme
+import com.epmedu.animeal.base.theme.DisabledButtonColor
+import com.epmedu.animeal.base.theme.DisabledButtonContentColor
+
+@Composable
+fun ShortButton(
+    text: String,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
```
Comment (egorikftp) [inline base/src/main/java/com/epmedu/animeal/base/ui/ShortButton.kt]: according to google guidlines modifier should be a first parameter

Comment (Fileek) [reply base/src/main/java/com/epmedu/animeal/base/ui/ShortButton.kt]: Sorry, but documentation says: "It should appear as the first optional parameter in the parameter list; **after all required parameters** (except for trailing lambda parameters) but before any other parameters with default values.",
text and onClick parameters are required, therefore modifier should be after them.

**Review by FlinForm — APPROVED** [review]

**Review by egorikftp — APPROVED** [review]

**Pull request merged**